CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # TRILLIAN Changelog
 
+## v1.0.7 - Storage API Changes, Schema Tweaks
+
+Published 2018-03-01 11:16:32 +0000 UTC
+
```

Comment:
I think we should have a section that explicitly summarizes changes to the gRPC APIs, as defined by the top-level .proto files.  Some of it is covered below (new log type) but there are other changes too, e.g.:

 - Add `InitLog` and `InitMap` methods for tree initialization.
 - Add `AddSequencedLeaf` and `AddSequencedLeaves` methods; these methods are still under development and may change in future versions.

Reply:
Done.